sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosTemplate.java

```diff
@@ -76,7 +79,7 @@
      */
     public CosmosTemplate(CosmosAsyncClient client, String databaseName,
                           CosmosConfig cosmosConfig, MappingCosmosConverter mappingCosmosConverter) {
-        this(new CosmosFactory(client, databaseName), cosmosConfig, mappingCosmosConverter);
+        this(new CosmosFactory(client, databaseName), cosmosConfig, mappingCosmosConverter, null);
```

Comment:
Because of multi-database support we have now - hard coding `null` in this constructor will be in-correct. 
As in that case, this auditing feature will not work in multi-database configuration. 

See `DatabaseConfiguration.java` class for more information on multi-database support example.

Reply:
added a new constructor that includes IsNewAwareAuditingHandler as a parameter for use in multi-database configuration. Updated sample in DatabaseConfiguration.java to use this new constructor with auditing enabled.